src/matter/cluster/WaterContentMeasurementCluster.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for WaterContent Measurement.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.3
+ */
+export const WaterContentMeasurementCluster = Cluster({
+    id: 0x0405,
```

Comment:
in fact we need 3 clusters with that ... because there are 3 IDs ... structure wise should be like LabelCluster maybe?